eva-accession-import-automation/run_accession_import.py

```diff
@@ -139,8 +140,10 @@ def main(command_line_args):
     parser = argparse.ArgumentParser(
         description='Run the accession import process for a given species, assembly and build', add_help=False,
         formatter_class=RawTextHelpFormatter)
-    parser.add_argument("-s", "--species", help="Species for which the process has to be run", required=True)
-    parser.add_argument("--scientific-name", help="Scientific name for the species (e.g. daucus_carota)", required=True)
+    parser.add_argument("-s", "--species", help="Species for which the process has to be run (e.g. red_sheep_469796"),
+                        required=True)
+    parser.add_argument("--scientific-name",
+                        help="Filesystem-friendly scientific name for the species (e.g. daucus_carota)", required=True)
```

Comment:
Could you align the example with the corresponding scientific name for red sheep?